drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -681,6 +687,7 @@ light_added = function(driver, device, parent_device_id, resource_id)
 
   driver.light_id_to_device[device_light_resource_id] = device
 
+  device:set_field(Fields.IS_ONLINE, true)
```

Comment:
Bulbs still are added if they are offline, correct? I think we should not emit any events until we know a device is online. In this case we will do the refresh for the device and emit initial state making the ST platform think the device is online when its not.

Reply:
I thought it would be better to have the field follow the synthetic state we establish in `added` to populate DW, and then let the SSE onopen code handle rectifying this.